Formula/keploy.rb

```diff
@@ -23,15 +23,24 @@ def install
     cp_r "public", "web", remove_destination: true
     system "go", "build", *std_go_args, "./cmd/server"
   end
+
   test do
+    require "pty"
+
     port = free_port
-    pid = fork do
-      assert_match("👍 connect to http://localhost:#{port}/ for GraphQL playground",
-                   shell_output("export PORT=#{port} && #{bin}/keploy"))
```

Comment:
This test is actually false positive as I did not see any `👍 connect to http://localhost:#{port}/ for GraphQL playground` in the server stdout